content/docs/rfcs/14/README.md

```diff
@@ -38,6 +38,10 @@ The `timestamp` field MAY be filled out to signify the time at which the message
 This field holds the Unix epoch time in nanoseconds. 
 Omitting it means the timestamp is unspecified.
 
+The `ephemeral` field MAY be filled out to signify the transient nature of the message.
+Omitting it means the message is permananent, and must be added to the store.
```

Comment:
I'd also put a link to the store protocol where it says:
> and must be added to the store.

Also, it makes it clearer to say `store protocol` instead of `store` as one may have no context about the store protocol when reading the WakuMessage specs.

Reply:
Addressed in [f7983f6](https://github.com/vacp2p/rfc/pull/532/commits/f7983f6f64459863e925478cad8d030bab50bf26)

and [69a8f7c](https://github.com/vacp2p/rfc/pull/532/commits/69a8f7cc719e2329b31ec320c69a234dbfe78c1c)